src/actions/app.js

```diff
@@ -2,6 +2,7 @@ export const UPDATE_TEXT_COLOR = 'UPDATE_TEXT_COLOR';
 export const BLUR_TEXT_COLOR = 'BLUR_TEXT_COLOR';
 export const UPDATE_FONT_SIZE = 'UPDATE_FONT_SIZE';
 export const BLUR_FONT_SIZE = 'BLUR_FONT_SIZE';
+export const UPDATE_FONT_UNIT_OF_MEASURE = 'UPDATE_FONT_UNIT_OF_MEASURE'
```

Comment:
How about we change the terminology from "Font unit of measure" to "Font size unit"? It's shorter, and also directly correlates with "Font size". If you happy with this change, we'll need to do it across the whole PR, not just here.